crates/cairo-coverage/src/input/sierra_to_cairo_map.rs

```diff
@@ -30,25 +34,52 @@ pub struct SierraToCairoMap(HashMap<StatementIdx, StatementOrigin>);
 #[derive(Clone, Eq, PartialEq)]
 pub struct StatementOrigin {
     pub function_name: FunctionName,
-    pub file_location: FileLocation,
+    pub source_file_full_path: SourceFileFullPath,
     pub line_range: LineRange,
 }
 
 impl StatementOrigin {
     pub fn remove_virtual_file_prefix(&mut self) {
-        self.file_location = VIRTUAL_FILE_REGEX
-            .replace_all(&self.file_location, "")
-            .to_string();
+        self.source_file_full_path = SourceFileFullPath(
+            VIRTUAL_FILE_REGEX
+                .replace_all(&self.source_file_full_path.0, "")
+                .to_string(),
+        );
+    }
+}
+
+#[derive(Deserialize, Clone, Eq, PartialEq)]
+pub struct LineRange {
+    pub start: LineNumber,
+    pub end: LineNumber,
+}
+
+impl From<&SourceCodeSpan> for LineRange {
+    fn from(span: &SourceCodeSpan) -> Self {
+        Self {
+            start: span.start.line + LineNumber(1),
+            end: span.end.line + LineNumber(1),
```

Comment:
what's the cause for this specific offsets?

Reply:
Beacuse most editors are 1 based and annotaions are 0 based. Added comment about that

Reply:
![american-psycho-willem-dafoe](https://github.com/user-attachments/assets/2b10948b-0bd5-4591-acab-7988bbd01421)